snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
It looks like sshd-keygen on Fedora also calls ssh-keygen, but I agree we shouldn't emulate the OS tools. On Debian there's ```/var/lib/dpkg/info/openssh-server.postinst``` which creates the keys. I think the dpkg-reconfigure command is a much better way.